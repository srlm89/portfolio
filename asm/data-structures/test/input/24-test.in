ERASE actual/24-test.out
INSERT 24 9.473281 Deep Sea
PRINT_IN_ORDER actual/24-test.out
INSERT 1 7.342722 Pinocchio [1940] [720p]
INSERT 19 9.851359 CJ7   720p
REMOVE_ID 19
PRINT_IN_ORDER actual/24-test.out
INSERT 25 3.028933 The Italian Job [2003] [720p]
INSERT 22 8.676745 Defiance.1080p
INSERT 18 6.377669 Aladdin [1992] [720p]
REMOVE_ID 1
PRINT_IN_ORDER actual/24-test.out
INSERT 16 6.430769 Angels & Demons [2009] [1080p]
INSERT 30 8.803741 Battlestar Galactica The Plan [720p]
INSERT 4 3.082419 Blood.Diamond.2006.1080p.HDDVD.x264-DEFiNiTE
INSERT 9 4.405595 James Bond 09 - The Man with the Golden Gun [1974]
REMOVE_ID 30
REMOVE_ID 9
PRINT_IN_ORDER actual/24-test.out
INSERT 15 2.196193 Kung Fu Panda Secrets of the Furious Five    720p
INSERT 7 2.439128 Sixth Sense   720p
INSERT 33 3.430290 Breach 1080p
INSERT 2 3.679421 Die Hard 4   1080p
INSERT 37 2.126083 Burn E [720p]
REMOVE_ID 7
REMOVE_ID 16
PRINT_IN_ORDER actual/24-test.out
INSERT 11 3.371664 007.Die Another Day.avi
INSERT 31 1.300333 Final Destination 3   1080p
INSERT 23 1.232252 Species   720p
INSERT 0 9.037816 Constantine 720p
INSERT 26 1.867013 Bedtime.Stories.720p.BluRay.x264-REFiNED
INSERT 5 1.242347 Mission Impossible Trilogy   720p
REMOVE_ID 37
REMOVE_ID 15
REMOVE_ID 5
PRINT_IN_ORDER actual/24-test.out
INSERT 28 0.254976 Batman Dark Knight   1080p
INSERT 32 3.589165 Chocolate 720p
INSERT 14 1.612918 James Bond 01 - Dr. No [1962] [720p]
INSERT 45 3.858654 The.Godfather.Part.II.1974.720p.BluRay.x264-ESiR_tehPARADOX.COM
INSERT 8 7.344405 Fearless - 720p
INSERT 20 6.582959 Big Nothing   720p
INSERT 3 3.050769 Dead Race 720p
REMOVE_ID 45
REMOVE_ID 28
REMOVE_ID 3
PRINT_IN_ORDER actual/24-test.out
INSERT 6 5.873466 Chicken Little 1080p
INSERT 17 6.141213 The X Files - I Want To Believe    720p
INSERT 50 3.831532 SAW III   720p
INSERT 48 8.108989 Fifty Dead Men Walking   1080p
INSERT 27 5.809228 Diary.Of.A.Sex.Addict.2008.720p.BluRay.x264-FLHD
INSERT 34 6.475535 Final Destination 4   1080p
INSERT 57 0.538085 Public Enemies [720p]
INSERT 59 9.546947 The.Godfather.Part.III.1990.720p.BluRay.x264-SiNNERS_tehPARADOX.COM
REMOVE_ID 14
REMOVE_ID 50
REMOVE_ID 20
REMOVE_ID 4
PRINT_IN_ORDER actual/24-test.out
INSERT 38 9.448016 The Haunting In Connecticut [2009] [720p]
INSERT 55 7.537274 Ip Man [2009]    720p
INSERT 51 1.591246 Horseman    720p
INSERT 12 6.600208 contents.txt
INSERT 54 1.305092 Back to the Future Part I [1985] [BluRay] [720p] [AC3] [EN]
INSERT 42 4.872623 Twilight [2008]   720p
INSERT 29 6.340899 Kill Shot   720p
INSERT 52 5.122663 Erin Brockovich 2000.720p
INSERT 40 5.469835 Eagle Eye    1080p
REMOVE_ID 26
REMOVE_ID 18
REMOVE_ID 55
REMOVE_ID 40
PRINT_IN_ORDER actual/24-test.out
