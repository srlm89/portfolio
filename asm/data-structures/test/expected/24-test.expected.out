[ 24 9.473281 Deep Sea ]
[ 1 7.342722 Pinocchio [1940] [720p] ][ 24 9.473281 Deep Sea ]
[ 18 6.377669 Aladdin [1992] [720p] ][ 22 8.676745 Defiance.1080p ][ 24 9.473281 Deep Sea ][ 25 3.028933 The Italian Job [2003] [720p] ]
[ 4 3.082419 Blood.Diamond.2006.1080p.HDDVD.x264-DEFiNiTE ][ 16 6.430769 Angels & Demons [2009] [1080p] ][ 18 6.377669 Aladdin [1992] [720p] ][ 22 8.676745 Defiance.1080p ][ 24 9.473281 Deep Sea ][ 25 3.028933 The Italian Job [2003] [720p] ]
[ 2 3.679421 Die Hard 4   1080p ][ 4 3.082419 Blood.Diamond.2006.1080p.HDDVD.x264-DEFiNiTE ][ 15 2.196193 Kung Fu Panda Secrets of the Furious Five    720p ][ 18 6.377669 Aladdin [1992] [720p] ][ 22 8.676745 Defiance.1080p ][ 24 9.473281 Deep Sea ][ 25 3.028933 The Italian Job [2003] [720p] ][ 33 3.430290 Breach 1080p ][ 37 2.126083 Burn E [720p] ]
[ 0 9.037816 Constantine 720p ][ 2 3.679421 Die Hard 4   1080p ][ 4 3.082419 Blood.Diamond.2006.1080p.HDDVD.x264-DEFiNiTE ][ 11 3.371664 007.Die Another Day.avi ][ 18 6.377669 Aladdin [1992] [720p] ][ 22 8.676745 Defiance.1080p ][ 23 1.232252 Species   720p ][ 24 9.473281 Deep Sea ][ 25 3.028933 The Italian Job [2003] [720p] ][ 26 1.867013 Bedtime.Stories.720p.BluRay.x264-REFiNED ][ 31 1.300333 Final Destination 3   1080p ][ 33 3.430290 Breach 1080p ]
[ 0 9.037816 Constantine 720p ][ 2 3.679421 Die Hard 4   1080p ][ 4 3.082419 Blood.Diamond.2006.1080p.HDDVD.x264-DEFiNiTE ][ 8 7.344405 Fearless - 720p ][ 11 3.371664 007.Die Another Day.avi ][ 14 1.612918 James Bond 01 - Dr. No [1962] [720p] ][ 18 6.377669 Aladdin [1992] [720p] ][ 20 6.582959 Big Nothing   720p ][ 22 8.676745 Defiance.1080p ][ 23 1.232252 Species   720p ][ 24 9.473281 Deep Sea ][ 25 3.028933 The Italian Job [2003] [720p] ][ 26 1.867013 Bedtime.Stories.720p.BluRay.x264-REFiNED ][ 31 1.300333 Final Destination 3   1080p ][ 32 3.589165 Chocolate 720p ][ 33 3.430290 Breach 1080p ]
[ 0 9.037816 Constantine 720p ][ 2 3.679421 Die Hard 4   1080p ][ 6 5.873466 Chicken Little 1080p ][ 8 7.344405 Fearless - 720p ][ 11 3.371664 007.Die Another Day.avi ][ 17 6.141213 The X Files - I Want To Believe    720p ][ 18 6.377669 Aladdin [1992] [720p] ][ 22 8.676745 Defiance.1080p ][ 23 1.232252 Species   720p ][ 24 9.473281 Deep Sea ][ 25 3.028933 The Italian Job [2003] [720p] ][ 26 1.867013 Bedtime.Stories.720p.BluRay.x264-REFiNED ][ 27 5.809228 Diary.Of.A.Sex.Addict.2008.720p.BluRay.x264-FLHD ][ 31 1.300333 Final Destination 3   1080p ][ 32 3.589165 Chocolate 720p ][ 33 3.430290 Breach 1080p ][ 34 6.475535 Final Destination 4   1080p ][ 48 8.108989 Fifty Dead Men Walking   1080p ][ 57 0.538085 Public Enemies [720p] ][ 59 9.546947 The.Godfather.Part.III.1990.720p.BluRay.x264-SiNNERS_tehPARADOX.COM ]
[ 0 9.037816 Constantine 720p ][ 2 3.679421 Die Hard 4   1080p ][ 6 5.873466 Chicken Little 1080p ][ 8 7.344405 Fearless - 720p ][ 11 3.371664 007.Die Another Day.avi ][ 12 6.600208 contents.txt ][ 17 6.141213 The X Files - I Want To Believe    720p ][ 22 8.676745 Defiance.1080p ][ 23 1.232252 Species   720p ][ 24 9.473281 Deep Sea ][ 25 3.028933 The Italian Job [2003] [720p] ][ 27 5.809228 Diary.Of.A.Sex.Addict.2008.720p.BluRay.x264-FLHD ][ 29 6.340899 Kill Shot   720p ][ 31 1.300333 Final Destination 3   1080p ][ 32 3.589165 Chocolate 720p ][ 33 3.430290 Breach 1080p ][ 34 6.475535 Final Destination 4   1080p ][ 38 9.448016 The Haunting In Connecticut [2009] [720p] ][ 42 4.872623 Twilight [2008]   720p ][ 48 8.108989 Fifty Dead Men Walking   1080p ][ 51 1.591246 Horseman    720p ][ 52 5.122663 Erin Brockovich 2000.720p ][ 54 1.305092 Back to the Future Part I [1985] [BluRay] [720p] [AC3] [EN] ][ 57 0.538085 Public Enemies [720p] ][ 59 9.546947 The.Godfather.Part.III.1990.720p.BluRay.x264-SiNNERS_tehPARADOX.COM ]
